Makefile

```diff
@@ -84,11 +84,11 @@ test:
 	@$(GPGME_ENV) GO111MODULE="on" go test $(BUILDFLAGS) -cover ./...
 
 fmt:
-	@go fmt -l -s -w $(SOURCE_DIRS)
+	@gofmt -l -s -w $(SOURCE_DIRS)
 
 validate: lint
 	@GO111MODULE="on" go vet ./...
-	@test -z "$$(go fmt -s -l . | grep -ve '^vendor' | tee /dev/stderr)"
+	@test -z "$$(gofmt -s -l . | grep -ve '^vendor' | tee /dev/stderr)"
```

Comment:
Thanks!

Not the topic of this PR, but I have also noticed https://github.com/containers/image/issues/1252 . Is there a canonical / elegant way to deal with that? `set  -o pipefail` seems supported by both Bash and Zsh, but is not POSIX.

```sh
full=$(gofmt -s -l . ) && echo "$full" | grep …
```
?

Reply:
One option is to move these commands into actual bash scripts where the environment/expectations can be more tightly controlled.  Normally not done for small one-liners like this but it would be easy to maintain.

Another option is to [explicitly set `SHELL` and `.SHELLFLAGS` in the Makefile](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html).  However that would then apply to EVERY shell command, which could have unintended consequences.